libs/sdl/sdl.c

```diff
@@ -427,6 +475,14 @@ HL_PRIM void HL_NAME(win_resize)(SDL_Window *win, int mode) {
 
 
 HL_PRIM void HL_NAME(win_swap_window)(SDL_Window *win) {
+#if TARGET_OS_IOS || TARGET_OS_TV
+	SDL_SysWMinfo info;
+	SDL_VERSION(&info.version);
+	SDL_GetWindowWMInfo(win, &info);
+
+	glBindFramebuffer(GL_FRAMEBUFFER, info.info.uikit.framebuffer);
+	glBindRenderbuffer(GL_RENDERBUFFER,info.info.uikit.colorbuffer);
+#endif

```

Comment:
I think should be done at the Haxe GlDriver level, by exporting SDL_GetWindowWMInfo. But maybe it's a bit more complex to do, so if it's reliable enough we can leave it as it.

Reply:
I think it would be really easier to leave it in the SDL C code right now. Yes, it is working fine on all iOS / tvOS devices.

Reply:
Is that not a duplicate of the ZIDX()==0 checks ?